modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/io/TaskWriter.java

```diff
@@ -219,6 +224,9 @@ public void writeEntityBody(Task item, JsonGenerator jg) throws IOException {
 
                 jg.writeEndObject();
             }
+        } catch (DocumentNotFoundException e) {
+            // If the DocumentRoute is not found (deleted for example), the TaskWriter fails.
+            log.error(e);
```

Comment:
If you want to log the stack trace you need `log.error(e, e)`.
If you don't, please add a comment saying so because it's surprising to hide it.

Reply:
Yes no special reason to hide the stack! Fixed!